files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,106 @@
+#54= IFCPROPERTYSETTEMPLATE('0cSUJcwWT3vQ6YDhD8WTjC',#9,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBoreHole',(#53));
+#55= IFCTUNNEL('2$M74xvK5FOvGEnGoq1S65',#9,'Tunnel name',$,$,#36,$,$,.ELEMENT.,.NOTDEFINED.);
+#56= IFCTUNNELPART('1bwZbg3L96EOUSRvBkBbRu',#9,'TunnelPart name',$,$,#36,$,$,.ELEMENT.,.NOTDEFINED.,.NOTDEFINED.);
+#57= IFCLOCALPLACEMENT(#36,#35);
+#58= IFCALIGNMENT('0yaE5fcMn1Zhp1MvVkuX5K',#9,'Name',$,$,#57,$,$);
+#59= IFCLOCALPLACEMENT(#57,#35);
+#60= IFCCARTESIANPOINTLIST2D(((0.,0.),(30.,0.),(45.,-15.),(60.,0.),(45.,15.)),$);
+#61= IFCINDEXEDPOLYCURVE(#60,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,2))),.F.);
+#62= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#24,$,.MODEL_VIEW.,$);
+#63= IFCSHAPEREPRESENTATION(#62,'Axis','Curve2D',(#61));
+#64= IFCPRODUCTDEFINITIONSHAPE($,$,(#63));
+#65= IFCALIGNMENTHORIZONTAL('3nMWevyOD289SYM1aXH__V',#9,'Name',$,$,#59,#64);
+#66= IFCCARTESIANPOINTLIST3D(((0.,0.,0.),(30.,0.,-1.),(45.,-15.,-2.),(60.,0.,0.),(45.,15.,2.),(30.,0.,1.)),$);
+#67= IFCINDEXEDPOLYCURVE(#66,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,6))),.F.);
+#68= IFCSHAPEREPRESENTATION(#62,'Axis','Curve3D',(#67));
+#69= IFCPRODUCTDEFINITIONSHAPE($,$,(#68));
+#70= IFCALIGNMENTVERTICAL('3Sspe6h7X22gs3KYqvgDuK',#9,'Name',$,$,#59,#69);
+#71= IFCCARTESIANPOINT((0.,0.));
+#72= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#71,0.,0.,0.,30.,$,.LINE.);
+#73= IFCALIGNMENTSEGMENT('1sODiIzcvFUvzazSAuaVgW',#9,'Name','Description',$,$,$,#72);
+#74= IFCCARTESIANPOINT((30.,0.));
+#75= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#74,4.71238898038469,15.,15.,47.1238898038469,$,.CIRCULARARC.);
+#76= IFCALIGNMENTSEGMENT('0cVjLJFu9EguI7Oxham0YS',#9,'Name','Description',$,$,$,#75);
+#77= IFCCARTESIANPOINT((60.,0.));
+#78= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#77,1.5707963267949,15.,15.,47.1238898038469,$,.CLOTHOID.);
+#79= IFCALIGNMENTSEGMENT('1yYmkMLjnF8Rjzkzuya2mN',#9,'Name','Description',$,$,$,#78);
+#80= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',0.,30.,0.,-0.0333333333333333,-0.0333333333333333,$,.CONSTANTGRADIENT.);
+#81= IFCALIGNMENTSEGMENT('2CUSA$1aLFKvy$elXBmjJR',#9,'Name','Description',$,$,$,#80);
+#82= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',30.,47.1238898038469,-1.,-0.0424413181578388,0.0848826363156775,-15.0084776206116,.CIRCULARARC.);
+#83= IFCALIGNMENTSEGMENT('23Oh1zFv58qwWyaQeh2cL_',#9,'Name','Description',$,$,$,#82);
+#84= IFCALIGNMENTVERTICALSEGMENT('StartTag','EndTag',77.1238898038469,47.1238898038469,0.,0.0848826363156776,-0.0424413181578387,-15.0084776206116,.PARABOLICARC.);
+#85= IFCALIGNMENTSEGMENT('3TuEcmTe59HB0aJaU2E4zx',#9,'Name','Description',$,$,$,#84);
+#86= IFCREFERENT('3w2rXXoxHDWxnYwiSBdsYw',#9,'Referent name',$,$,#36,$,.BOUNDARY.);
+#87= IFCGROUP('0Iqm6uyOjA3xdYjjWz_1gw',#9,'Referent Group',$,$);

```

Comment:
Maybe not a requirement in this sprint, but I think that all groups (incl systems) should be declared using the [Project Declaration](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/project-declaration.htm) template.